environments/echis/postgresql.yml

```diff
@@ -1,6 +1,8 @@
-override:
+pgbouncer_override:
```

Comment:
I think it's worth keeping in mind that if once we have people we can't speak to directly managing their own instances, we have to be very careful about changing the format of the environment files. And when we do, we'll have to create a changelog outlining how someone else needs to update their environments, to fit the change, and update the docs (http://dimagi.github.io/commcare-cloud/commcare-cloud/env/#postgresqlyml) for that file.

Reply:
yes for sure. I did consider that and though about trying some other method like splitting by var prefix but that seemed a bit hacky.

I will definitely put together a changelog for this.